webpack.config.js

```diff
@@ -205,16 +210,43 @@ module.exports = [
         },
         performance: getPerformanceHints(5 * 1024)
     }),
+
+    // Because both video-blur-effect and rnnoise-processor modules are loaded
+    // in a lazy matter using the loadScript function with a hard coded name,
+    // i.e.loadScript('libs/rnnoise-processor.min.js'), webpack dev server
+    // won't know how to properly load them using the default config filename
+    // and sourceMapFilename parameters which target libs without .min in dev
+    // mode. Thus we change these modules to have the same filename in both
+    // prod and dev mode.
     Object.assign({}, config, {
         entry: {
             'video-blur-effect': './react/features/stream-effects/blur/index.js'
         },
         output: Object.assign({}, config.output, {
             library: [ 'JitsiMeetJS', 'app', 'effects' ],
-            libraryTarget: 'window'
+            libraryTarget: 'window',
+            filename: '[name].min.js',
+            sourceMapFilename: '[name].min.map'
         }),
         performance: getPerformanceHints(1 * 1024 * 1024)
     }),
+
+    Object.assign({}, config, {
+        entry: {
+            'rnnoise-processor': './react/features/stream-effects/rnnoise/index.js'
+        },
+        output: Object.assign({}, config.output, {
+            library: [ 'JitsiMeetJS', 'app', 'effects' ],
+            libraryTarget: 'window',
+            filename: '[name].min.js',
+            sourceMapFilename: '[name].min.map'
+        }),
+        performance: getPerformanceHints(4 * 1024 * 1024)
+
+    }),
+
+    // The Webpack configuration to bundle external_api.js (aka
+    // JitsiMeetExternalAPI).
```

Comment:
I guess this came up in a rebase? Can you please drop it?

Reply:
Dropped.